pkgs/development/python-modules/osqp/default.nix

```diff
@@ -12,11 +12,14 @@
 
 buildPythonPackage rec {
   pname = "osqp";
-  version = "0.6.1";
+  version = "0.6.2-2020-03-18";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "130frig5bznfacqp9jwbshmbqd2xw3ixdspsbkrwsvkdaab7kca7";
+  src = fetchgit {
+    url = "https://github.com/GuillaumeDesforges/osqp-python";
+    rev = "770ecc3b96a94b30344cf3721a2e0771ef5c8259";
+    # branch = "pr/update-version";
+    fetchSubmodules = true;
+    sha256 = "0rzx1sqdgih42wjyymilcm6hj5hihq3rnjacflmlr5i14wsjpric";
```

Comment:
I would just apply the patch instead, and just mark when it can be removed

Reply:
Will do

Reply:
It comes from the fact that 0.6.1 had issues and that apparently they solved it, but did not release the fixes. I was hoping that they'd do so (and release a 0.6.2). So it's not just a bump a version that my PR gives, because it also carries the latest fixes from upstream.

Reply:
if that is the case, the version should be `unstable-YYYY-MM-DD` where the date is taken from the commit date

Reply:
I don't see a PR on the upstream https://github.com/oxfordcontrol/osqp/pulls?q=is%3Aopen+is%3Apr from you.

Reply:
It is here: https://github.com/oxfordcontrol/osqp-python/pull/42

They will "release a new minor version soon". (https://github.com/oxfordcontrol/osqp-python/pull/42#issuecomment-602060826)